data/themes/darktable.css

```diff
@@ -253,44 +253,36 @@ box *
   color: @disabled_fg_color;
 }
 
-/* Sets outer borders that could be hide or shown in darktable with 'b' shortcut.
-  remember, in gui/gtk.c, outer borders (active areas used to collapse panels)
-  are defined to 5 px × DPI. We cheat by setting inner borders/spacing here to 5 px as well. */
+/* Sets outer borders that could be hide or shown in darktable with 'b' shortcut */
 #outer-border
 {
   background-color: @border_color;
-  min-width: 20px;
-}
-#resize-border
-{
-  background-color: #ff0000;
+  min-width: 1.4em;
 }
 
 /* Default gtk buttons */
 button,
 #add-color-button
 {
-  padding: 2px;
-  border: 1px solid @button_border;
+  padding: 0.14em;
+  border: 0.07em solid @button_border;
   background-color: @button_bg;
   color: @button_fg;
   font-weight: normal;
   font-family: sans-serif;
-  min-height: 1em;
-  min-width: 1em;
-  padding: 2px;
+  padding: 0.14em;
 }
 
 button:disabled,
 #add-color-button:disabled
 {
-  border: 1px solid shade(@button_border, 0.95);
+  border: 0.07em solid shade(@button_border, 0.95);
 }
 
 /* Default extra margin for icons optical alignment */
 #button-canvas
 {
-  margin: 5px; /* not real pixels, this is used as a percent extra space */
+  margin: 0.35em; /* not real pixels, this is used as a percent extra space */
```

Comment:
based on the comment I think perhaps this one needs to be left alone doesn't it?

Reply:
Gtk apply pixels from em depending on font size, so should not be a problem. On what I test, it's all good. I see no issue. I've tested with 11px font size and your small font size, 8.4px. But feel free to test too, I could have missed something.

Reply:
My recollection was that somewhere (in the overlays maybe) the number of pixels was converted to a percentage in gtk. It may be that code has been removed now - perhaps @AlicVB can recall, but if that comment is not applicable any longer it should be removed.

Reply:
As I know, it's always pixels. But depending on how it sets, that do something like a percentage. Just check with Gtk Inspector Tool, and with margin set to 5px, you will see 5px. With 0.35em with 11px, you will see 5.0xxx px. But on those settings, that is considered to some percentage by adjusting space remaining depending on margin set, not real percentage. I don't know if I'm clear here. So, having 5px or 0.35em doesn't change what that is for. The only change is that we have the logic (not real pixels and used as a "percent" extra space) but scalable.

Reply:
I've just tested, and yes, the order seems : 
- margin is defined in css with em
- when retrieve by gtk functions, we get the corresponding pixels (internally computed by gtk)
- then we use that value as a percentage of the thumb size

So I don't any problem here (it's more an enhancement) apart from the comment which is not accurate anymore. Maybe something like `/* not real value, corresponding pixels size is used as a percent extra space */` but I'm sure there's better English speakers than me around ;)

Reply:
Thanks @AlicVB. Is that the same for everywhere this comment appears in darktable.css? Comment should perhaps be something along the lines of `/* this em value is converted to pixels by Gtk and then used as a percentage of extra space around the element */`.

Reply:
Thanks for the suggestion @elstoc. Seems indeed more appropriate. Will update that soon.

Reply:
comment updated with your proposal @elstoc.